envoy/api/v2/core/health_check.proto

```diff
@@ -116,6 +117,16 @@ message HealthCheck {
     string service_name = 1;
   }
 
+  // [#not-implemented-hide:] Custom health check.
+  message ExtensionHealthCheck {
+    // The registered name of the extension health check.
+    string name = 1 [(validate.rules).string.min_bytes = 1];
+
+    // An extension health checker specific configuration which depends on the extension health
```

Comment:
per above "A custom health checker..." (check for other uses of extension below)

Reply:
Addressed in 55b3672. Thanks!